### PR TITLE
analytics instrumentation done

### PR DESCRIPTION
**Pull request by dselans**
Need to clean a few things up and test a bit more.

Short version:

User will be presented with an interactive config prompt that looks like the attached img.

image

The user will be presented with the reconfig prompt under the following conditions:

* If they've never had the prompt.
* If their plumber's major version != version saved in config.
* Don't have a config at all
* Have a TTY

Seems to be working well!

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -24,6 +25,11 @@ import (
 	"github.com/batchcorp/plumber/prometheus"
 )
 
+const (
+	// Note: Only used if analytics are enabled (default: no)
+	MixPanelToken = "2d2995d51084155edc043905f2dc528f"
```
Comment (blinktag) [inline main.go:30]: Should probably be injected via build tags

File: options/options.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	VERSION = "UNSET"
+	VERSION = "v0.2.2"
```
Comment (blinktag) [inline options/options.go:25]: Undo

Comment (dselans) [reply options/options.go]: TY!

File: main.go (hunk 1 above)
Comment (dselans) [reply main.go:31]: 👍

**Review by blinktag — APPROVED** [review]

**Pull request merged**